doc/Makefile

```diff
@@ -7,5 +7,9 @@
 SUBDIR+=	doc
 .endif
 
+.if ${.MAKE.LEVEL} == 0
```

Comment:
Just occurred to me that i could _combine_ this condition with the one in the previous `.if` using an _or_. Should i do that?

Reply:
No opinion about the _or_, either way is fine.

However unfortunately `.MAKE.LEVEL` doesn't seem to exist for openbsd's make(1), which we target. Can you find an equivalent way to do this that works for the common subset of bmake and openbsd make?
https://man.openbsd.org/make